packages/legacy-client/src/client/actions/createActionProxy.ts

```diff
@@ -123,3 +128,45 @@ export function createBulkActionProxy<
   ) as BulkActions<O>;
   return proxy;
 }
+
+export function createBatchActionProxy<
```

Comment:
Is this identical to the createBulkActionProxy?

Reply:
Its pretty much the same, but inside of `createBulkActionProxy` it's checking for all of the Bulk types e.g. `Op extends BulkActionExecutionOptions`, `WrappedBulkActionReturnType`, etc. and returns as `BulkActions<O>` rather than `BatchActions<O>` which are all the same I guess, but I figured I should still just keep them separate